sw-emulator/lib/periph/src/root_bus.rs

```diff
@@ -265,6 +266,9 @@ pub struct CaliptraRootBus {
     #[peripheral(offset = 0x1002_8000, mask = 0x0000_7fff)]
     pub sha256: HashSha256,
 
+    #[peripheral(offset = 0x1003_0000, mask = 0x0000_7fff)] // TODO update when known
+    pub dilithium: Dilithium,
```

Comment:
Minor nit: as you pointed out there are some small changes from dilithium to ML-DSA. I think we should consider naming this peripheral something like mldsa87. But I'm not sure what it's named in RTL.

cc @mhatrevi @mojtaba-bisheh